core/boot.js

```diff
@@ -346,11 +351,14 @@ async function bootGhost() {
         // Step 2 - Start server with minimal app in global maintenance mode
         debug('Begin: load server + minimal app');
         const rootApp = require('./app');
-        const GhostServer = require('./server/ghost-server');
-        ghostServer = new GhostServer({url: config.getSiteUrl()});
-        await ghostServer.start(rootApp);
-        bootLogger.log('server started');
-        debug('End: load server + minimal app');
+
+        if (withBackend) {
+            const GhostServer = require('./server/ghost-server');
+            ghostServer = new GhostServer({url: config.getSiteUrl()});
+            await ghostServer.start(rootApp);
+            bootLogger.log('server started');
+            debug('End: load server + minimal app');
```

Comment:
Little thing but this debug statement shouldn't be caught by the if statement

Reply:
A more hairy comment 😬 

IMO mounting the backend onto the app, and booting the server are not the same thing. I wouldn't expect setting backend/withBackend false to result in the server not being created 🤔 I'd expect it to result in a server without any backend paths mounted.